res/controllers/Reloop Ready.midi.xml

```diff
@@ -0,0 +1,1983 @@
+                <midino>0x00</midino>
+                <options>
+                    <script-binding/>
+                </options>
+            </control>
+            <control>
+                <group></group>
+                <key>ReloopReady.components.leftDeck.fxUnit.fxButtons[0].input</key>
+                <status>0x88</status>
+                <midino>0x00</midino>
+                <options>
+                    <script-binding/>
+                </options>
+            </control>
+            <control>
+                <group></group>
+                <key>ReloopReady.components.leftDeck.fxUnit.fxButtons[1].input</key>
+                <status>0x98</status>
+                <midino>0x01</midino>
+                <options>
+                    <script-binding/>
+                </options>
+            </control>
+            <control>
+                <group></group>
+                <key>ReloopReady.components.leftDeck.fxUnit.fxButtons[1].input</key>
+                <status>0x88</status>
+                <midino>0x01</midino>
+                <options>
+                    <script-binding/>
+                </options>
+            </control>
+            <control>
+                <group></group>
+                <key>ReloopReady.components.leftDeck.fxUnit.fxButtons[2].input</key>
+                <status>0x98</status>
+                <midino>0x02</midino>
+                <options>
+                    <script-binding/>
+                </options>
+            </control>
+            <control>
+                <group></group>
+                <key>ReloopReady.components.leftDeck.fxUnit.fxButtons[2].input</key>
+                <status>0x88</status>
+                <midino>0x02</midino>
+                <options>
+                    <script-binding/>
+                </options>
+            </control>
+            <control>
+                <group></group>
+                <key>ReloopReady.components.leftDeck.fxUnit.loop.knob.input</key>
+                <status>0xB4</status>
+                <midino>0x34</midino>
+                <options>
+                    <script-binding/>
+                </options>
+            </control>
+            <control>
+                <group></group>
+                <key>ReloopReady.components.leftDeck.fxUnit.loop.button.input</key>
+                <status>0x94</status>
+                <midino>0x40</midino>
+                <options>
+                    <script-binding/>
+                </options>
+            </control>
+            <control>
+                <group></group>
+                <key>ReloopReady.components.leftDeck.fxUnit.loop.button.input</key>
+                <status>0x84</status>
+                <midino>0x40</midino>
+                <options>
+                    <script-binding/>
+                </options>
+            </control>
+
+
+
+            <control>
+                <group></group>
+                <key>ReloopReady.components.leftDeck.fxUnit.level.input</key>
```

Comment:
```suggestion
                <key>ReloopReady.components.rightDeck.fxUnit.level.input</key>
```
Left deck fx is already in line 281 and there is no rightDeck.fxUnit.level.input anywhere, so knob is not working